system-addon/content-src/styles/_activity-stream.scss

```diff
@@ -84,7 +78,7 @@ a {
   padding: 15px 25px 0;
 
   button {
-    background-color: $input-secondary;
+    background: var(--newtab-button-secondary-color);
```

Comment:
Was there a need to make this `background` and not more scoped to `background-color`? It would probably be better consistency / expectation to use `background-color` wherever we use a `var(…color)`

Reply:
no need, just saving bytes 😝 . kind of my personal preference but dont really care either way.